docs/PacketFence_CoovaChilli_Quick_Integration_Guide.asciidoc

```diff
@@ -0,0 +1,209 @@
+
+That step is fairly simple to accomplish and is a one time deal.
+
+In the root account credentials section, enter root as Username and click 'Test'. You'll be prompted for a new root password. Choose a password for the MySQL root user and click 'Save'. You can now enter your newly setted password in the root account credentials section and click 'Test'.
+
+Next section of this step is the PacketFence user account on the MySQL server. Simply leave the default pf username here and choose of a password. This one will automatically be set in the PacketFence configuration where you'll be able to retrive it in any case. Once the password entered twice, click 'Create user'.
+
+Third section will create the database and load the correct schema on it. Simply leave the default and click 'Create tables and indexes'.
+
+Step 4: PacketFence Configuration
++++++++++++++++++++++++++++++++++
+This step will configure the general options of your PacketFence installation.
+
+Almost all of the required information here are self-explanatory. The only one that could be confusing is the DHCP Servers section. In this one, enter a comma-delimited list of all the DHCP Server on the customer network so when PacketFence will see DHCP trafic originating
+ from these IPs, no rogue-dhcp alerts will be triggered. It can safely be ignored for the purpose of this guide.
+
+Step 5: Administration
+++++++++++++++++++++++
+This is the step where we create the administrative user to access the PacketFence Administration Web Interface.
+
+Simply provide the desired username and password, then click 'Create user'.
+
+Step 6: Services & Confirmation
++++++++++++++++++++++++++++++++
+The last but not the least. Here, we start the PacketFence server according to the configurations made in the previous steps. If everything goes as expected, you'll be prompted by a window inviting you to continue to the web administration interface.
+
+You'll be able to login to the PacketFence web administration interface with the credentials created in Step 5.
+
+Services status will help you monitor if everything goes as expected. If not, you'll see which service is in trouble and the log output will help you determine the problem that occurs.
+
+Step 2: Access Point and CoovaChilli Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Now that we have a functionnal PacketFence installation, we will go ahead and start by configuring the access point and CoovaChilli running on it. We will then use some of the configuration parameters to finish PacketFence integration in step 3.
+
+This guide assume that CoovaChilli is installed on the access point. If it's not, we suggest you Google around and make sure to find the appropriate way to do so since there's just to much vendor and network equipment that supports CoovaChilli to document this step here.
```

Comment:
... we suggest you search relevant information on the Internet to install CoovaChilli as there are too many network equipment vendors that support CoovaChilli to accurately document this step here.